apps/myaccount/src/components/profile/profile.tsx

```diff
@@ -636,11 +659,12 @@ export const Profile: FunctionComponent<ProfileProps> = (props: ProfileProps): R
                                                             onClick={ () => {
                                                                 setShowMobileUpdateWizard(true);
                                                             } }
-                                                            onKeyPress={ (e) => {
-                                                                if (e.key === "Enter") {
-                                                                    setShowMobileUpdateWizard(true);
-                                                                }
-                                                            } }
+                                                            onKeyPress={
+                                                                (e: React.KeyboardEvent<HTMLAnchorElement>) => {
+                                                                    if (e.key === "Enter") {
```

Comment:
Can `e` be undefined? else lets destructure from params `({key}: Type)`

Reply:
First part is fixed by https://github.com/wso2/identity-apps/pull/3763/commits/25d7975004f07604b9d6adb6dd1e18041b8ef502

About that latter part, if we remove the `React` import, it will break other places as well right?